operator/internal/manifests/gateway_tenants.go

```diff
@@ -55,7 +55,11 @@ func ApplyGatewayDefaultOptions(opts *Options) error {
 	return nil
 }
 
-func configureGatewayDeploymentForMode(d *appsv1.Deployment, mode lokiv1.ModeType, fg configv1.FeatureGates, stackName, stackNs string) error {
```

Comment:
AFAICS we configure the TLS minVersion and ciphers only for OpenShift here. However we support per definition of the feature gate this feature for non-openshift tenants too. Thus this feature gate impact on the lokistack-gateway and opa-openshift container does not belong here. IMHO this belongs in the `manifests/gateway.go` eg as we configure the HTTP PKI for example.

Reply:
I see. I must have misunderstood what the object was.